packages/jupyter-ai/jupyter_ai/tests/test_directory.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test that the `collect_filepaths` function only selects files that are
+1. Not in the the excluded directories and
+2. Are in the  valid file extensions list.
+"""
```

Comment:
This is still essentially a comment and not a docstring. Please see relevant example in the docs at the bottom of the linked paragraph: https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
This has been corrected and checked.